packages/wrangler/src/__tests__/helpers/run-wrangler.ts

```diff
@@ -11,7 +11,11 @@ export async function runWrangler(
 ) {
 	const originalEnv = process.env;
 	try {
-		process.env = { ...originalEnv, ...env };
+		process.env = {
+			...originalEnv,
+			...env,
+			VITEST_CACHE_BUST: new Date().toISOString(),
```

Comment:
can you explain the need for this here?

Reply:
Wasn't sure if there was a more preferred way of solving this...
In zones.ts, I created a global variable to cache zone/ID mappings.
A couple tests were failing because that global variable does not get reset between tests, leading to incorrect results being returned.

Reply:
This would not be an issue in actual usage because wrangler isn't deploying multiple times in a row, and (more importantly) an account/zone combo will always return the same ID from the API (at least within the lifetime of a wrangler process)